testsuite/testsuite.go

```diff
@@ -84,7 +84,7 @@ func Runtime() (context.Context, *runtime.Runtime) {
 		Config:            cfg,
 	}
 
-	logrus.SetLevel(logrus.DebugLevel)
+	slog.SetDefault(slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelDebug})))
```

Comment:
doesn't this fail to set it for logging lines which haven't been converted yet?

Reply:
Right, let me revert this line, we'll change that when we are completely removing logrus